docs/tutorials/advanced/geonode_production/geosites.txt

```diff
@@ -84,6 +83,32 @@ Permissions by Site
 ===================
 By default GeoNode is publicly available. In the case of GeoSites, new data will be publicly available, but only for the site it was added to, and the master site (all data is added to the master site). 
 
+Activate geosites app
+================
+In order to use geonode in Multi-tenancy mode, follow these steps:
+1. check in settings.py if 'geonode.contrib.geosites' in GEONODE_CONTRIB_APPS is rightly uncommented
+2. add in settings.py 'geonode.contrib.geosites' in INSTALLED_APPS
+3. run python manage.py syncdb
+
 Adding New Sites
 ================
-A management command exists to easily create a new site.  This will create all the needed directories, as well as a site specific settings file.  The command may also create a website configuration file.
+To add a new site follow the following steps:
+
+1. copy the directory site_template in /geonode/geonode/contrib/geosites and give it a name
+2. from geonode administration pannel add 'new site'
+3. create a virtualhost in the webserver related to the new created site. Remember to setup the WSGIDeamonProcess with the name you gave to the folder created at point 1. and the path to the geosites directory. WSGIProcessGroup have to be pointed to the name you choose for the folder you created at point 1. Eventually, WSGIScriptAlias have to be set to the wsgi.py you have in your site folder.
+4. check the configuration files: local_settings.py, pre_settings.py, post_settings.py in /geonode/geonode/contrib/geosites as well as local_settings.py and settings.py in your site folder:
+    - in /geonode/geonode/contrib/geosites/local_settings.py set the variable SERVE_PATH. it has to point geosites folder.
+    - in the local_setting of the site folder insert the values to the following variables: 
+            * SITE_ID
+            * SITE_NAME
+            * SITE_URL
+6. create static_root directory in your geosites project directory (e.g., /geonode/geonode/contrib/geosites/static_root) and give it grants:
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/thumbs
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/layers
+    sudo chown geonode:www-data ../geonode/geonode/contrib/geosites/uploaded/
+    sudo chmod -Rf 777 ../geonode/geonode/contrib/geosites/uploaded/thumbs
```

Comment:
777 is not good permission setting, giving ownership to www-data is usually enough. for uploaded/layers the apache configuration should prevent access as well.